drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -152,6 +149,8 @@ static void GLAPIENTRY _gl_debug_print(GLenum source, GLenum type, GLuint id, GL
 		strcpy(debSource, "Application");
 	} else if (source == _EXT_DEBUG_SOURCE_OTHER_ARB) {
 		strcpy(debSource, "Other");
+	} else {
+		ERR_FAIL_MSG(vformat("GL ERROR: Invalid or unhandled source '%d' in debug callback.", source));
```

Comment:
These fix these errors:
![image](https://github.com/godotengine/godot/assets/4701338/09dc7171-ed5a-4972-bb03-3b32c4b6438a)

CC @lawnjelly

Reply:
Although I'm in the blame this is because I'm registered as initial author for 4.x, but this was copy pasted from 3.x GLES2, which is karrofel it looks like.

But fix looks fine to me. :+1: